app/assets/javascripts/backbone/views/project_step_view.coffee

```diff
@@ -68,7 +69,20 @@ class MS.Views.ProjectStepView extends Backbone.View
         MS.timelineView.addBlankStep() unless @persisted || @duplicate
       else
         $('#calendar-step-modal').modal('hide')
-    
+
     else if $(e.target).is('a.action-delete')
       @$el.remove()
     MS.calendarView.refresh()
+
+  showLogModal: (e) ->
+    e.preventDefault()
+    link = e.currentTarget
+    action = @$(link).data('action')
+
+    if !@modalView
+      @modalView = new MS.Views.LogModalView()
```

Comment:
Always use `unless` in this case, and can use the one-line variant here `x unless y`